src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/FabDropDown.cs

```diff
@@ -0,0 +1,90 @@
+using System;
```

Comment:
Is there any other differences between this and the existing dropdown rendering?
If no, can we reuse the existing dropdown instead of creating a new class?

Reply:
I add a new **FabDropDown** class and change the type of parameter **dropdownView** from **DropDown** to **FabDropDown**  in **AddDropDown** method 

Also in **Line 81** of FabDropDown rendering class, I pass a new Rendering Type(FabDropDown) when create a instance DiagnosticData Class. 
```
var diagData = new DiagnosticData()
            {
                Table = table,
                RenderingProperties = new Rendering(RenderingType.FabDropdown)
```

so that in detector code, we can instantiate between **DropDown** class and **FabDropDown** class to render different dropdown model
```
//render old dropdown
DropDown dropdownViewModel = new DropDown("dropdown model",data);
res.AddDropdownView(dropdownViewModel ,"Drop Down Title");

//render Fabric UI dropdown
FabDropDown fabDropdownViewModel = new FabDropDown("fab dropdown model",data);
res.AddDropdownView(fabDropdownViewModel ,"Fab Drop Down Title");
```